### ISSUE TITLE
CelebA GAN training not working as expected

### ISSUE THREAD

**Issue by oscarwernqvist** [aidotse/LeakPro#303]
# Issue

The GAN training in the CelebA example is not working as expected, as found by Fazelehs testing

## Problem Description

We've found the following problems that needs to be adressed: 

- In train_gan() in celebA_plgmi_handler.py, the discriminator is shown augmented fake images - they are not augmented in the original paper.
- In plgmi.py, the current version of where train_gan() is called, the incorrect batch size is passed. 
- In celebA_data.py, the labels are assigned in "alphabetical" order instead of numerical order making it difficult to relate to the original CelebA training data set.  

## Expected Behavior

The example should match the GAN training method of the original PLG-MI attack

## What Needs to be Done

- Change train_gan() method in handler accordingly
- Change train_gan() call in plgmi.py accordingly
- Change celebA_data.py to la

## How Can It Be Tested or Reproduced

Run the attack and verify that the GAN training is working.
